src/test/resources/Features/OCD-2023-monitorDownloadFiles.feature

```diff
@@ -1,39 +1,47 @@
-      | STG  |    2011 | xml  |   92 |  148 | MB    |  3885 |
-      | STG  |    2014 | xml  |    1 |  450 | MB    |  5190 |
-      | STG  |    2014 | csv  |    1 | 2900 | KB    |  5190 |
-      | STG  |    2015 | xml  |    1 |  150 | MB    |   590 |
-      | STG  |    2015 | csv  |    1 |  350 | KB    |   590 |
-      | PROD |    2011 | xml  |   92 |  148 | MB    |  3885 |
-      | PROD |    2014 | xml  |    1 |  450 | MB    |  5190 |
-      | PROD |    2014 | csv  |    1 | 2900 | KB    |  5190 |
-      | PROD |    2015 | xml  |    1 |  150 | MB    |   590 |
-      | PROD |    2015 | csv  |    1 |  350 | KB    |   590 |
+Scenario Outline: The download files are "recent" 
+    Given I am on download the CHPL resources page on "<env>" 
+    When I download the "<edition>" "<type>" products file 
+    Then the download file is no more than "<days>" days old 
+    And the download file is at least "<size>" "<units>" in size 
+    And the download file has at least "<items>" items 
+    Examples: 
+        | env  | edition | type | days | size | units | items |
+        | DEV  |    2011 | xml  |   92 |  148 | MB    |  3885 |
+        | DEV  |    2014 | xml  |    1 |  450 | MB    |  5190 |
+        | DEV  |    2014 | csv  |    1 | 2900 | KB    |  5190 |
+        | DEV  |    2015 | xml  |    1 |  150 | MB    |   590 |
+        | DEV  |    2015 | csv  |    1 |  350 | KB    |   590 |
+        | STG  |    2011 | xml  |   92 |  148 | MB    |  3885 |
+        | STG  |    2014 | xml  |    1 |  450 | MB    |  5190 |
+        | STG  |    2014 | csv  |    1 | 2900 | KB    |  5190 |
+        | STG  |    2015 | xml  |    1 |  150 | MB    |   590 |
+        | STG  |    2015 | csv  |    1 |  350 | KB    |   590 |
+        | PROD |    2011 | xml  |   92 |  148 | MB    |  3885 |
+        | PROD |    2014 | xml  |    1 |  450 | MB    |  5190 |
+        | PROD |    2014 | csv  |    1 | 2900 | KB    |  5190 |
+        | PROD |    2015 | xml  |    1 |  150 | MB    |   590 |
+        | PROD |    2015 | csv  |    1 |  350 | KB    |   590 |
 
-  Scenario: At any given time, when downloaded, Surveillance Activity file is up-to-date
-    When I download the Surveillance Activity file
-    Then the downloaded file shows surveillance-all.csv filename
+Scenario Outline: 
+    At any given time, when downloaded, file name ends with Date and time format 
+    When I select the "<File>", download it from drop-down and I see the "<Downloaded File Name>" 
+    Then the "<Downloaded File Name>" ends with "yyyyMMdd_HHmmss" 
+    Examples: 
+        |File                 |Downloaded File Name             |
+        |Surveillance Activity|surveillance-all                 |
+        |Non-Conformities     |surveillance-with-nonconformities|
 
-  Scenario: At any given time, when downloaded, Non-Conformities file is up-to-date
-    When I download the Non-Conformities file
-    Then the downloaded file shows surveillance-with-nonconformities.csv filename
+Scenario: 
```

Comment:
I have this as a separate scenario because the chpl-sed file is on the Collections page whereas the other files are on the Resources page. 
Yes, it's the same logic!